src/main/java/leetcode2/GoodPairsSolutionTest.java

```diff
@@ -0,0 +1,20 @@
+package leetcode2;
```

Comment:
Please, move all your test classes to appropriate directories src/test/java/
e.g.  GoodPairsSolutionTest must be placed in src/test/java/leetcode2